agent-plane/agent-plane-protocol/src/main/java/org/eclipse/tractusx/agents/edc/SkillStore.java

```diff
@@ -22,10 +22,11 @@
 /**
  * interface to a skill store
  */
-public interface ISkillStore {
```

Comment:
What is the convention for Interfaces in KA? Not to have it with a leading capital I ?

Reply:
Relevant is the tractus-x convention/style guide. A leading I is no more allowed, instead the interface has a normal name and the implementaiton could be named SkillStoreImpl or so.